.github/workflows/test-backend.yml

```diff
@@ -0,0 +1,47 @@
+name: Test Backend
+on: 
+  pull_request:
```

Comment:
was it part of this ticket to look into the `path` option here?

Reply:
ermm..... yes it was 😃 

Updated with a path filter so it only runs for changes with the `backend` folder.